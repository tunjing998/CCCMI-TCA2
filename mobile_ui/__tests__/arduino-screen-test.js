import React, {Component} from 'react';
import {View, Text} from 'react-native';
import ArduinoScreen from '../screens/ArduinoScreen';
import ArduinoScreen2 from '../screens/ArduinoScreen2';
import renderer from 'react-test-renderer';

test('ArduinoScreen snapShot', () => {
  const snap = renderer.create(<ArduinoScreen />).toJSON();
  expect(snap).toMatchSnapshot();
});

test('ArduinoScreen2 snapShot', () => {
  const snap = renderer.create(<ArduinoScreen2 />).toJSON();
  expect(snap).toMatchSnapshot();
});
