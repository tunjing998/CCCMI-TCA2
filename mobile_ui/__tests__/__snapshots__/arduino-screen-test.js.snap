// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArduinoScreen snapShot 1`] = `
<View
  accessibilityLabel="arduinoScreenContainer"
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
  testID="arduinoScreenContainer"
>
  <Text
    style={
      Object {
        "color": "rgb(28, 28, 30)",
        "fontSize": 28.285714285714285,
        "paddingBottom": 70,
      }
    }
  >
    Connect to Arduino.
  </Text>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "flexDirection": "row",
        "justifyContent": "center",
        "marginBottom": 30,
      }
    }
  >
    <TextInput
      accessibilityLabel="arduinoScreenIDTextInput"
      allowFontScaling={true}
      placeholder="Insert Arduino ID here."
      rejectResponderTermination={true}
      style={
        Object {
          "backgroundColor": "#fff",
          "borderBottomColor": "rgb(28, 28, 30)",
          "borderBottomWidth": 1,
          "color": "#424242",
          "flex": 1,
          "marginLeft": 10,
          "paddingBottom": 10,
          "paddingLeft": 0,
          "paddingTop": 10,
        }
      }
      testID="arduinoScreenIDTextInput"
      underlineColorAndroid="transparent"
    />
    <View
      accessibilityLabel="arduinoScreenSearchIconButton"
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "overflow": "hidden",
          },
          Object {
            "backgroundColor": "transparent",
            "borderRadius": 5,
          },
        ]
      }
      testID="arduinoScreenSearchIconButton"
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "justifyContent": "flex-start",
              "padding": 8,
            },
            Object {
              "backgroundColor": "transparent",
              "borderRadius": 5,
            },
            undefined,
          ]
        }
      >
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": "#000",
                "fontSize": 20,
              },
              Object {
                "marginRight": 10,
              },
              Object {
                "fontFamily": "Material Design Icons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
        >
          Û∞çâ
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`ArduinoScreen2 snapShot 1`] = `
<View
  accessibilityLabel="arduinoConnectScreenContainer"
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
  testID="arduinoConnectScreenContainer"
>
  <Text
    style={
      Object {
        "color": "#242424",
      }
    }
  >
    asd
  </Text>
</View>
`;
